ep04_CMS_Higgs_Search.ipynb

```diff
@@ -7,7 +7,7 @@
         "colab_type": "text"
       },
       "source": [
-        "<a href=\"https://colab.research.google.com/github/plttraining/hsf_matplotlib_notebooks/blob/main/Higgs_4leptons.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
+        "<a href=\"https://colab.research.google.com/github/amangoel185/hsf_matplotlib_notebooks/blob/amangoel185%2Ffix%2Frestructure/ep04_CMS_Higgs_Search.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
```

Comment:
We can probably change the links after merging, or else someone with the access to `plttraining` would have to manually add the notebooks